### PR TITLE
Added organizational influence

### PR DESCRIPTION
**Pull request by zhongjun2**

### CODE REVIEW

File: chinese/wg-value/focus-areas/organizational-value/organizational-influence.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+###  这个指标是候选基线指标。评审这个指标请看[问题 #146](https://github.com/chaoss/wg-value/issues/146)。根据评论周期，这个指标将在下一个常规版本发布。
```
Comment (eyehwan) [inline chinese/wg-value/focus-areas/organizational-value/organizational-influence.md:1]: we could remove this line, no translations needed.

File: chinese/wg-value/focus-areas/organizational-value/organizational-influence.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+# 组织影响
+
+问题: 一个组织对一个社区有多大影响?
+
+## 描述
+
+组织影响力是衡量一个组织对开源社区的影响力的一个尺度。组织，通常是较大的企业公司，可能会影响项目的方向，表明在社区中有某种程度的控制。这相当于一个组织对其参与并拥有既得利益的开源项目的开发轨迹的影响程度。
```
Comment (eyehwan) [inline chinese/wg-value/focus-areas/organizational-value/organizational-influence.md:9]: remove space

File: chinese/wg-value/focus-areas/organizational-value/organizational-influence.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+## 目标
+
+组织能在开源社区中产生影响。对组织影响力的衡量本身就具有启发性，也可以作为一个显示组织影响力随时间增长的机会。这个指标可以帮助组织中的开源倡导者证明继续为参与提供资金和支持是合理的。该指标还可以帮助开源维护人员跟踪和测量组织的影响，以增加其项目的合法性，并监控各个组织的控制水平。组织影响力也可以作为一个信号，表明新成员为开源项目做出贡献可能是容易的，也可能是困难的。
```
Comment (eyehwan) [inline chinese/wg-value/focus-areas/organizational-value/organizational-influence.md:13]: remove space

File: chinese/wg-value/focus-areas/organizational-value/organizational-influence.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
+###  这个指标是候选基线指标。评审这个指标请看[问题 #146](https://github.com/chaoss/wg-value/issues/146)。根据评论周期，这个指标将在下一个常规版本发布。
+
+# 组织影响
```
Comment (eyehwan) [inline chinese/wg-value/focus-areas/organizational-value/organizational-influence.md:3]: 组织影响力

File: chinese/wg-value/focus-areas/organizational-value/organizational-influence.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+* 组织成员的级别和类型 [贡献者类型]  (https://chaoss.community/metric-types-of-contributions/)
+* 以与社区活动水平成比例的高比率提交变更请求的组织。例如，其他贡献组织的一个标准差或更少。有关 [组织多样性](https://chaoss.community/metric-organizational-diversity/)
+* 技术指导委员会的组织成员
+* 理事会组织成员
+* 项目维护者角色中的组织成员
+* 赞助开源项目的组织
+
+## 可视化
+
+### 提供指标的工具
```
Comment (eyehwan) [inline chinese/wg-value/focus-areas/organizational-value/organizational-influence.md:30]: 提供度量的工具

File: chinese/wg-value/focus-areas/organizational-value/organizational-influence.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+
+组织能在开源社区中产生影响。对组织影响力的衡量本身就具有启发性，也可以作为一个显示组织影响力随时间增长的机会。这个指标可以帮助组织中的开源倡导者证明继续为参与提供资金和支持是合理的。该指标还可以帮助开源维护人员跟踪和测量组织的影响，以增加其项目的合法性，并监控各个组织的控制水平。组织影响力也可以作为一个信号，表明新成员为开源项目做出贡献可能是容易的，也可能是困难的。
+
+## 实现
+
+在衡量组织影响力时要考虑的一些具体例子包括：
+Some specific examples to consider when measuring organizational influence include: 
+
+* 有多少组织成员为一个项目做出贡献 - [贡献者](https://chaoss.community/metric-contributors/)
+* 组织成员的级别和类型 [贡献者类型]  (https://chaoss.community/metric-types-of-contributions/)
```
Comment (eyehwan) [inline chinese/wg-value/focus-areas/organizational-value/organizational-influence.md:21]: [贡献者类型] (https://chaoss.community/metric-types-of-contributions/)

File: chinese/wg-value/focus-areas/organizational-value/organizational-influence.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+组织影响力是衡量一个组织对开源社区的影响力的一个尺度。组织，通常是较大的企业公司，可能会影响项目的方向，表明在社区中有某种程度的控制。这相当于一个组织对其参与并拥有既得利益的开源项目的开发轨迹的影响程度。
+
+## 目标
+
+组织能在开源社区中产生影响。对组织影响力的衡量本身就具有启发性，也可以作为一个显示组织影响力随时间增长的机会。这个指标可以帮助组织中的开源倡导者证明继续为参与提供资金和支持是合理的。该指标还可以帮助开源维护人员跟踪和测量组织的影响，以增加其项目的合法性，并监控各个组织的控制水平。组织影响力也可以作为一个信号，表明新成员为开源项目做出贡献可能是容易的，也可能是困难的。
+
+## 实现
+
+在衡量组织影响力时要考虑的一些具体例子包括：
+Some specific examples to consider when measuring organizational influence include:
```
Comment (eyehwan) [inline chinese/wg-value/focus-areas/organizational-value/organizational-influence.md:18]: remove above line content

File: chinese/wg-value/focus-areas/organizational-value/organizational-influence.md (hunk 4 above)
Comment (zhongjun2) [reply chinese/wg-value/focus-areas/organizational-value/organizational-influence.md]: Done

File: chinese/wg-value/focus-areas/organizational-value/organizational-influence.md (hunk 1 above)
Comment (zhongjun2) [reply chinese/wg-value/focus-areas/organizational-value/organizational-influence.md]: Done

File: chinese/wg-value/focus-areas/organizational-value/organizational-influence.md (hunk 5 above)
Comment (zhongjun2) [reply chinese/wg-value/focus-areas/organizational-value/organizational-influence.md]: Done

File: chinese/wg-value/focus-areas/organizational-value/organizational-influence.md (hunk 6 above)
Comment (zhongjun2) [reply chinese/wg-value/focus-areas/organizational-value/organizational-influence.md]: Done

File: chinese/wg-value/focus-areas/organizational-value/organizational-influence.md (hunk 7 above)
Comment (zhongjun2) [reply chinese/wg-value/focus-areas/organizational-value/organizational-influence.md]: Done

**Comment by eyehwan** [comment]
Looks good to me .

**Pull request merged**